docker-compose.yaml

```diff
@@ -1,3 +1,5 @@
+version: '3.8'
```

Comment:
@pcheremu do we need a version here? I would rather not add a version if there is no reason for it.

Reply:
@vasyl-ivanchuk Yes, it does not work on CI without a version. I have tried again https://github.com/matter-labs/block-explorer/actions/runs/7889736145/attempts/1?pr=46